protocol/metrics/metrics_consumer_manager.go

```diff
@@ -28,14 +28,16 @@ type ConsumerMetricsManager struct {
 	LatestBlockMetric                      *prometheus.GaugeVec
 	LatestProviderRelay                    *prometheus.GaugeVec
 	virtualEpochMetric                     *prometheus.GaugeVec
+	apiSpecificsMetric                     *prometheus.GaugeVec
```

Comment:
Why not `apiMethodCalls`? You've already called the new field in `RelayMetrics`: `ApiMethod`